backend/oasst_backend/utils/hugging_face.py

```diff
@@ -1,10 +1,20 @@
+from enum import Enum
 from typing import Any, Dict
 
 import aiohttp
 from oasst_backend.config import settings
 from oasst_shared.exceptions import OasstError, OasstErrorCode
 
 
+class HF_url(str, Enum):
```

Comment:
this may have already been here, but the class names here mix camel- and snake case

Reply:
good, point Classes/Enums should always be UpperCamelCase. Unfortunately we are dealing here with two abbreviations, e.g. HF and URL ... depending on how we want to handle them it could be `HfUrl`, `HFUrl` or `HFURL`... since we have `OasstApiClient` . `HFURL` can be ruled out .. I would suggest to use `HfUrl` ...